src/components/diagrams/singleLineDiagram/single-line-diagram.js

```diff
@@ -326,6 +328,17 @@ const SingleLineDiagram = forwardRef((props, ref) => {
     const [svgFinalWidth, setSvgFinalWidth] = useState();
     const [svgFinalHeight, setSvgFinalHeight] = useState();
 
+    useEffect(() => {
+        if (finalPaperHeight) {
+            setDisplayedSldHeights((displayedSldHeights) => {
+                return [
+                    ...displayedSldHeights.filter((sld) => sld.id !== sldId),
+                    { id: sldId, initialHeight: finalPaperHeight },
+                ];
+            });
+        }
+    }, [finalPaperHeight, setDisplayedSldHeights, sldId]);
+
     useLayoutEffect(() => {
         const sizes = computePaperAndSvgSizesIfReady(
```

Comment:
Something like setSvgFinalHeight(computedHeight); could work for all the pane to be navigable with the SVG

Reply:
It was causing visual glitches if I implemented it through setSvgFinalHeight, I found a way by directly setting the svg height attribute line 589